main.go

```diff
@@ -115,9 +116,15 @@ func main() {
 		"The address the health endpoint binds to.")
 	flag.IntVar(&webhookPort, "webhook-port", 9443,
 		"Webhook Server port (set to 0 to disable)")
+	opts := zap.Options{
+		Development: true,
```

Comment:
Should we set this based on `devLogging` instead of hard coding it?
```suggestion
		Development: devLogging,
```